app/views/questionnaire.py

```diff
@@ -271,7 +271,7 @@ def _save_sign_out(collection_id, eq_id, form_type, this_location, form):
 
 def _household_answers_changed(answer_store):
     household_answers = answer_store.filter(block_id="household-composition")
-    if len(household_answers) != len(request.form)-1:
+    if len(household_answers) != len(request.form)-2:
```

Comment:
Yeh i might make this more explicit by copying the dict and removing the expected answers so the lengths will be equal?